packages/@react-spectrum/illustrated-message/src/IllustratedMessage.tsx

```diff
@@ -32,36 +32,41 @@ function IllustratedMessage(props: SpectrumIllustratedMessageProps, ref: DOMRef<
     illustrationProps,
     headingProps
   } = useIllustratedMessage(props);
+  let headingClassName = classNames(
```

Comment:
Is there a way to refactor this to not have both typographyStyles and IllustratedMessage styles?

Reply:
There's a large typography upgrade that needs to happen. @dannify do we have that on our roadmap? for now I think you can just keep doing it the way it was

Reply:
No it isn't but I am concerned about including all of typography in this one component. It is a bit of an overkill. 
You can keep it as is for now.